### PR TITLE
Major rework

### PR DESCRIPTION
**Pull request by gpotter2**
- **Scapy terminal**
- new download section
- better mobile support
- move to SVG images (from fonts)

Demo over https://gpotter2.github.io/secdev.github.io/

### CODE REVIEW

File: src/assets/scapy-2.5.0.dev177-py3-none-any.whl
Comment (guedou) [inline src/assets/scapy-2.5.0.dev177-py3-none-any.whl:1]: That is a huge file. Is there something that we can do to make it smaller?

**Review by guedou — APPROVED** [review]
Awesome PR!

File: src/assets/scapy-2.5.0.dev177-py3-none-any.whl
Comment (gpotter2) [reply src/assets/scapy-2.5.0.dev177-py3-none-any.whl:1]: We could generate it in the CI. Would that work?

Comment (guedou) [reply src/assets/scapy-2.5.0.dev177-py3-none-any.whl:1]: I had a look at the current `.tar.gz` assuming that the wheel content is similar. Stripping the file down means to remove `contrib/` and `tls/`:
```shell
du -h -d 1 scapy/layers/tls/ scapy/contrib/
168K	scapy/layers/tls//crypto
640K	scapy/layers/tls/
160K	scapy/contrib//scada
104K	scapy/contrib//isotp
1.2M	scapy/contrib//automotive
 56K	scapy/contrib//rtps
3.3M	scapy/contrib/
```

**Comment by gpotter2** [comment]
Wheel is now generated automatically. Please
# **squash and merge**

File: src/assets/scapy-2.5.0.dev177-py3-none-any.whl
Comment (gpotter2) [reply src/assets/scapy-2.5.0.dev177-py3-none-any.whl:1]: It's only 1.4MB. I'd argue it's fine. I'm also now generating it automatically, and stripping it down sounds like a pain.

**Comment by gpotter2** [comment]
I've added support for file upload because it's a bit cool. Should be ready to merge !

**Pull request merged**